src/devices/fancy-node/fancy-node.cc

```diff
@@ -147,35 +186,77 @@ void setup() {
     led_manager->FatalErrorAnimation();
   }
 
+  analogReadResolution(10);
+
+  bool button_still_pressed = !(digitalRead(kButton1) && digitalRead(kButton2));
+  if (button_pressed && button_still_pressed) {
```

Comment:
Why not always just show the battery on boot?

Reply:
To read the battery we need to delay by a couple of seconds, to let the filter capacitor charge up. I think it's annoying to have a delay on boot, so I made this only activate if a button is pressed.